docs/articles/documentation/guides/basic-guides/run-tests.md

```diff
@@ -663,3 +665,15 @@ await runner
 > Note that quarantine mode increases the test task's duration because failed tests are executed three to five times.
 
 See Martin Fowler's [Eradicating Non-Determinism in Tests](http://martinfowler.com/articles/nonDeterminism.html) article for more information about non-deterministic tests.
+
+## Troubleshooting
+
+### Disable the 'Close Multiple Tabs' Popup in IE
+
+Internet Explorer displays a native dialog when you try to close a browser window with more than one tab opened. This dialog blocks the page and halts tests. Disable this popup before you run tests.
```

Comment:
Does Internet Explorer has non native dialogs? I guess, it doesn't.

 `Internet Explorer displays a dialog...`

`the page and halts tests` -> `the page and stops/freezes tests`
Use the simplest vocabulary, because a lot of our users are not native English speakers.

Reply:
```suggestion
Internet Explorer displays a dialog when you try to close a browser window with more than one tab opened. This dialog blocks the page and pauses tests. Disable this popup before you run tests.
```